[All Hosts] (yo office) new overview of the Yeoman generator tool (#3299)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-debugging.md

## Requirements

> This debugging process works **only** for UI-less custom functions, which don't use a task pane or other UI elements. A UI-less custom function can be created by following the steps in the [Create custom functions in Excel](../tutorials/excel-tutorial-create-custom-functions.md) tutorial, and then removing all of the task pane and UI elements that are installed by the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).

Reviewer: ```suggestion
This debugging process works **only** for UI-less custom functions, which don't use a task pane or other UI elements. To create a UI-less custom function, follow the steps in the [Create custom functions in Excel](../tutorials/excel-tutorial-create-custom-functions.md) tutorial, and then remove all of the task pane and UI elements that are installed by the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
```
